pre_publish.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+
+./util/update_lints.py
+
+git status --short | sort | grep -v README.md | grep -v helper.txt > helper.txt
+
+# abort if the files differ
+diff "publish.files" "helper.txt"
+
+rm helper.txt
+
+# add all changed files
+git add .
+git commit -m "version bump"
```

Comment:
This definitely needs to contain the version then. [Also, this fails rules 3 and 5.](https://chris.beams.io/posts/git-commit/)